ale_linters/haskell/ghc_mod.vim

```diff
@@ -0,0 +1,23 @@
+" Author: wizzup <wizzup@gmail.com>
+" Description: ghc-mod for Haskell files
+
+call ale#Set('haskell_ghc_mod_executable', 'ghc-mod')
+
+function! ale_linters#haskell#ghc_mod#GetCommand (buffer) abort
+    let l:executable = ale#Var(a:buffer, 'haskell_ghc_mod_executable')
+
+    return ale#handlers#haskell_stack#EscapeExecutable(l:executable, 'ghc-mod')
+\       . ' --map-file %s=%t check %s'
```

Comment:
Put the line continuation character on the same level of indentation as the previous line.